src/main/java/org/opentripplanner/graph_builder/module/ned/ElevationModule.java

```diff
@@ -161,7 +169,7 @@ public void buildGraph(Graph graph, HashMap<Class<?>, Object> extra) {
 
         // Multithread elevation calculations
         ForkJoinPool forkJoinPool = new ForkJoinPool(
-            Runtime.getRuntime().availableProcessors(),
+            parallelism,
             forkJoinWorkerThreadFactory,
             null,
             false
```

Comment:
Do you know if creating a ThreadPool with just 1 thread will lead to a performance overhead compared with just running the Tasks in the current thread?

Reply:
Yes, it appears that the ThreadPool does have slightly worse performance with the HSL setup. Over 10 runs, the code that didn't use a ThreadPool was on average 9 seconds faster. I have added this code to this PR.